systemd-swap

```diff
@@ -165,36 +166,35 @@ case "$1" in
   start)
     [ -f "${LOCK_STARTED}" ] && ERRO "$0 already started"
     touch "${LOCK_STARTED}"
+
+    # load default values
+    . "${DEF_CONFIG}" || ERRO "Error loading ${ETC_CONFIG}"
+
     load_config_fragments(){
-      # The main configuration file is read before any of the configuration
-      # directories, and has the lowest precedence; entries in a file in any configuration
-      # directory override entries in the single configuration file.
+      # config precedence follows systemd scheme:
+      # etc > run > lib > swap.conf
```

Comment:
Should probably rewrite this to explicitly say `usr` or something instead of `swap.conf`